relatio/narrative_models.py

```diff
@@ -570,9 +585,14 @@ def label_clusters(
                     f"Multiple labels for cluster {clu}- 2 shown: {token_most_common}. First one is picked.",
                     RuntimeWarning,
                 )
-            self.labels_unknown_entities[index_clustering_model][
-                clu
-            ] = token_most_common[0][0]
+            if token_most_common[0][0] != "":
```

Comment:
Sounds very idiosyncratic and linked to a bad text cleaning upstream. Wouldn't hardcode this here.

Reply:
When I checked the narrative entity labels, I found a unnamed entity cluster label: "". So I wrote some codes to replace it with a not empty label.

You mean it is very specific and I don't have to add this. But instead, if "" appears in the the final narrative outputs, they should be ignored (like the same way as the tutorial.ipynb) ?

Reply:
Yup, let's leave this to the user